tags-precedence.md

```diff
@@ -17,5 +19,12 @@ As the Tag element is optional, a lot of `Matroska Readers` do not handle it and
 So for maximum compatibility, it's usually better to put the strings in the `TrackEntry`, `ChapterAtom` and `Attachment`
 and keep the tags matching these values if tags are also used.
 
-See [@?MatroskaTags] for more details on how to use tags.
+## Tag Levels
+
+Tag elements allow tagging information on multiple levels, each level having a `TargetTypeValue` (#targettypevalue-element).
+An element for a given `TargetTypeValue` also applies to the lower levels, ie smaller `TargetTypeValue` values. If an upper value
```

Comment:
`ie` > `i.e.`

Reply:
Agreed, and maybe even change the wording completely to something like "…also applies to lower levels denoted by smaller `TargetTypeValues` values."